codetable/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "multihash-codetable"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+blake2b_simd = { version = "1.0.0", default-features = false }
+blake2s_simd = { version = "1.0.0", default-features = false }
+blake3 = { version = "1.2.0", default-features = false }
+digest = { version = "0.10.1", default-features = false }
+sha-1 = { version = "0.10.0", default-features = false }
+sha-2 = { version = "0.10.0", default-features = false, package = "sha2" }
+sha-3 = { version = "0.10.0", default-features = false, package = "sha3" }
```

Comment:
Not sure if it should be part of this PR, I just want to mention it: `sha-1` is now named `sha1`, so we should just switch to the original package names without the dash.